### PR TITLE
Create api endpoints

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
### Project Requirements

- Create a new Rails API app called 'hello-rails-back-end'.
- Initialize your project with Git.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route).

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  namespace :api, defaults: { format: :json } do
+    namespace :v1 do
+      get 'random', to: 'greetings#random'
```
Comment (DeliceLydia) [inline config/routes.rb:10]: - [x] I am not able to fetch different greetings on the frontend, check if errors are not coming from not setting up `cors` well
cors

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Ugboaja-Uchechi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights are:
- Followed git workflow
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: config/routes.rb (hunk 1 above)
Comment (rloterh) [inline config/routes.rb:10]: - [x] Kindly endeavor to install and configure CORS gems so as to resolve this error, as recommended in the previous code review.

**Error:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Stephanie, 

You did great on this project, and for making the changes requested in the previous review(s)! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Please take your time to carefully improve aspects outlined in the previous code review(s), and please check to see that the changes implemented are working as intended._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @Ugboaja-Uchechi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

image

image

nodding-yes

## Optional suggestions

- [ ] N/A

#### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
#### _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**